api/serializers.py

```diff
@@ -169,6 +171,18 @@ def get_rating(self, event):
         rate = Rate.objects.filter(event=event).aggregate(Avg('rate'))
         return rate['rate__avg']
 
+    def get_current_user_rating(self, event):
+        request = self.context.get("request")
+        if request and not request.user.is_anonymous:
+            user = request.user
+            try:
+                user_rate = Rate.objects.get(event=event, user=user).rate
+            except Rate.DoesNotExist:
+                user_rate = None
+            return user_rate
+        else:
+            return None
```

Comment:
I think we can replace this whole block with
`return Rate.objects.filter(event=event, user=user).first()`

This would return the Rate instance if it exists (instead of only its score), or None if it doesn't.

Reply:
Nice! I'll take it, but the return type cannot be a rate instance, so I'll still use .first().rate

Reply:
DANGER, if it returns None, you will get an AttributeError